dotnet/src/Connectors/Connectors.AzureOpenAI/Settings/AzureOpenAITextToAudioExecutionSettings.cs

```diff
@@ -0,0 +1,130 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Microsoft.SemanticKernel.Text;
+
+namespace Microsoft.SemanticKernel.Connectors.AzureOpenAI;
+
+/// <summary>
+/// Execution settings for Azure OpenAI text-to-audio request.
+/// </summary>
+[Experimental("SKEXP0001")]
+public sealed class AzureOpenAITextToAudioExecutionSettings : PromptExecutionSettings
+{
+    /// <summary>
+    /// The voice to use when generating the audio. Supported voices are alloy, echo, fable, onyx, nova, and shimmer.
+    /// </summary>
+    [JsonPropertyName("voice")]
+    public string Voice
```

Comment:
Agree of Mark, optional for this too, and use it as a property, not sure whi this is a field, same for others.

```suggestion
    public string Voice { get; } = DefaultVoice;
```

Reply:
We'll update all properties of all new *PromptExecutionSettings classes in on go in scope of the issue - https://github.com/microsoft/semantic-kernel/issues/7111.